src/components/Navigation.tsx

```diff
@@ -464,6 +448,29 @@ const Navigation: FC = () => {
                     },
                   )}
                 >
+                  {isAuthenticated && (
+                    <li className="p-side-navigation__item">
+                      <div
+                        className="p-side-navigation__link"
+                        title={loggedInUserName}
```

Comment:
It might be useful to expose the ID as part of this title as well. This would be the email for OIDC users and the fingerprint for TLS certs. Something like this:

```suggestion
                        title={`${loggedInUserName} ${loggedInUserId}`}
```

Reply:
![image](https://github.com/canonical/lxd-ui/assets/58699506/07c5d4f6-f738-443f-bc28-0546275e934c)

Reply:
Yes, thanks for adding it.